src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/HtmlFieldTrumbowygEditorSettingsDriver.cs

```diff
@@ -38,11 +38,14 @@ public override async Task<IDisplayResult> UpdateAsync(ContentPartFieldDefinitio
                 var settings = new HtmlFieldTrumbowygEditorSettings();
 
                 await context.Updater.TryUpdateModelAsync(model, Prefix);
-                
+
                 try
                 {
-                    settings.Options = model.Options;
-                    JObject.Parse(settings.Options);
+                    if (!string.IsNullOrEmpty(model.Options))
+                    {
+                        settings.Options = model.Options;
+                        JObject.Parse(settings.Options);
```

Comment:
Is this needed?

Reply:
No, no longer needed after fixing the admin controller. I've removed it